src/pages/FileModal.tsx

```diff
@@ -123,7 +123,10 @@ const FileModal: React.FC<FileModalProps> = ({ file, onExited }) => {
           ))}
         </Select>
         {loadingFileLink ? (
+          <>
+         
           <CircularProgress />
+          </>
```

Comment:
are you sure you need this nested fragment here? You should be able to get rid of the `<> </>`